src/ray/common/task/task_spec.h

```diff
@@ -34,6 +34,8 @@ namespace ray {
 typedef ResourceSet SchedulingClassDescriptor;
 typedef int SchedulingClass;
 
+using RuntimeEnvHash = size_t;
```

Comment:
```suggestion
using RuntimeEnvHash = int64_t;
```

(no reason to use a platform-specific type?)